rts/Game/UI/KeyBindings.cpp

```diff
@@ -942,6 +945,9 @@ bool CKeyBindings::ExecuteCommand(const std::string& line)
 	if (buildHotkeyMap)
```

Comment:
Kinda related, it seems to me `buildHotkeyMap` is not being correctly set on certain operations. It should always rebuild the hotkeymap when any bindings have changed. So the hotkey map is not being updated when for example `unbindaction` is called.

This is not necessarily related to the scope of this PR, I can add an issue for it. But there's some possibility of reusing this variable to assert on whether any key has actually changed (we'd be able to avoid using `changedKeys` for example), which is the same logic you are using.